response/core/serializers.py

```diff
@@ -8,36 +8,60 @@
 from django.contrib.auth.models import User
 
 
-class ExternalUserSerializer(serializers.HyperlinkedModelSerializer):
-    owner = serializers.PrimaryKeyRelatedField(queryset=User.objects.all(), required=False)
-
+class ExternalUserSerializer(serializers.ModelSerializer):
     class Meta:
         model = ExternalUser
-        fields = ('app_id', 'external_id', 'owner', 'display_name')
+        fields = ("app_id", "external_id", "display_name")
 
 
-class ActionSerializer(serializers.HyperlinkedModelSerializer):
+class ActionSerializer(serializers.ModelSerializer):
     # Serializers define the API representation.
-    incident = serializers.PrimaryKeyRelatedField(queryset=Incident.objects.all(), required=False)
-    user = serializers.PrimaryKeyRelatedField(queryset=ExternalUser.objects.all(), required=False)
+    incident = serializers.PrimaryKeyRelatedField(
+        queryset=Incident.objects.all(), required=False
+    )
+    user = serializers.PrimaryKeyRelatedField(
+        queryset=ExternalUser.objects.all(), required=False
+    )
 
     class Meta:
         model = Action
-        fields = ('pk', 'details', 'done', 'incident', 'user')
+        fields = ("pk", "details", "done", "incident", "user")
 
 
-class IncidentSerializer(serializers.HyperlinkedModelSerializer):
-    reporter = serializers.PrimaryKeyRelatedField(queryset=ExternalUser.objects.all(), required=False)
-    lead = serializers.PrimaryKeyRelatedField(queryset=ExternalUser.objects.all(), required=False)
+class IncidentSerializer(serializers.ModelSerializer):
```

Comment:
I don't have context on why we were using a `HyperlinkedModelSerializer`, but this means we return hyperlinks to other API endpoints instead of primary keys. We don't need either in this serializer, so might as well use the simpler `ModelSerializer`